frontend/uddhar-frontend/src/shared/components/Navbar.jsx

```
@@ -26,6 +43,20 @@ const Navbar = ({children}) => {
                   <Link to={link.path}>{link.name}</Link>
                 </p>
               ))}
+              {isLoggedIn ? (
+                <p className="text-lg hover:text-gray-500">
+                  <button className="cursor-pointer" onClick={handleSignOut}>Sign Out</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)